geode-dunit/src/integrationTest/java/org/apache/geode/codeAnalysis/AnalyzeDUnitSerializablesIntegrationTest.java

```diff
@@ -16,13 +16,19 @@
 
 import org.junit.experimental.categories.Category;
 
+import org.apache.geode.test.dunit.internal.DUnitSanctionedSerializablesService;
 import org.apache.geode.test.junit.categories.SerializationTest;
 
-@Category({SerializationTest.class})
-public class AnalyzeCoreSerializablesJUnitTest extends AnalyzeSerializablesJUnitTestBase {
+@Category(SerializationTest.class)
+public class AnalyzeDUnitSerializablesIntegrationTest extends AnalyzeSerializablesJUnitTestBase {
```

Comment:
Note: github diff is confused... code is correct.

AnalyzeDUnitSerializablesIntegrationTest is a new test in geode-dunit.

AnalyzeCoreSerializablesJUnitTest was renamed to AnalyzeCoreSerializablesIntegrationTest which still exists in geode-core.